Tests/Remora.Discord.Commands.Tests/Extensions/CommandTreeExtensionTests.cs

```diff
@@ -730,6 +730,144 @@ public void CreatesMinAndMaxLengthConstrainedParametersCorrectly()
                 Assert.Equal(0u, parameter.MinLength.Value);
                 Assert.Equal(1u, parameter.MaxLength.Value);
             }
+
+            /// <summary>
+            /// Tests whether the method responds appropriately to a successful case.
+            /// </summary>
+            [Fact]
+            public void CreatesMinValueConstrainedParametersCorrectly()
+            {
+                var builder = new CommandTreeBuilder();
+                builder.RegisterModule<NumericalParameterWithValueConstraints.MinValueConstraint>();
+
+                var tree = builder.Build();
+
+                var result = tree.CreateApplicationCommands();
+                ResultAssert.Successful(result);
+
+                var commands = result.Entity;
+
+                var command = commands.Single();
+                var parameter = command.Options.Value.Single();
+
+                Assert.Equal(0l, parameter.MinValue.Value);
+                Assert.False(parameter.MaxValue.HasValue);
+            }
+
+            /// <summary>
+            /// Tests whether the method responds appropriately to a successful case.
+            /// </summary>
+            [Fact]
+            public void CreatesMaxValueConstrainedParametersCorrectly()
+            {
+                var builder = new CommandTreeBuilder();
+                builder.RegisterModule<NumericalParameterWithValueConstraints.MaxValueConstraint>();
+
+                var tree = builder.Build();
+
+                var result = tree.CreateApplicationCommands();
+                ResultAssert.Successful(result);
+
+                var commands = result.Entity;
+
+                var command = commands.Single();
+                var parameter = command.Options.Value.Single();
+
+                Assert.Equal(1l, parameter.MaxValue.Value);
+                Assert.False(parameter.MinValue.HasValue);
+            }
+
+            /// <summary>
+            /// Tests whether the method responds appropriately to a successful case.
+            /// </summary>
+            [Fact]
+            public void CreatesMinAndMaxValueConstrainedParametersCorrectly()
+            {
+                var builder = new CommandTreeBuilder();
+                builder.RegisterModule<NumericalParameterWithValueConstraints.MinAndMaxValueConstraint>();
+
+                var tree = builder.Build();
+
+                var result = tree.CreateApplicationCommands();
+                ResultAssert.Successful(result);
+
+                var commands = result.Entity;
+
+                var command = commands.Single();
+                var parameter = command.Options.Value.Single();
+
+                Assert.Equal(0l, parameter.MinValue.Value);
+                Assert.Equal(1l, parameter.MaxValue.Value);
```

Comment:
```suggestion
                Assert.Equal(1L, parameter.MaxValue.Value);
```

Reply:
addressed in [f6692a0](https://github.com/Remora/Remora.Discord/pull/235/commits/f6692a0b8af281b38ee0d9e2509409f58165c9a9)